rust/agama-server/src/network/web.rs

```diff
@@ -101,10 +101,10 @@ pub async fn network_service<T: Adapter + Send + Sync + 'static>(
                 .put(update_connection)
                 .get(connection),
         )
-        .route("/connections/:id/connect", get(connect))
-        .route("/connections/:id/disconnect", get(disconnect))
+        .route("/connections/:id/connect", patch(connect))
```

Comment:
see below reasoning, but I think that post is better here then patch.

Reply:
See https://github.com/openSUSE/agama/pull/1519#discussion_r1714878241

If you don't mind, we can postpone this discussion until @teclator and @imobachgs are back. As far as I know, this was a change agreed by them. So we can merge as it is and change our mind later if needed.